tests/unit-test.js

```diff
@@ -181,3 +182,24 @@ describe('Util:', () => {
     });
   });
 });
+
+describe('Index', async () => {
+  fixture.writeSync('tests', {
+    'add.js':`exports.add = (a, b) => {
+      return a + b;
+    }`
+  });
+  describe('runTest', () => {
+    it('test passes', async (assert) => {
```

Comment:
mocha supports promises, there is no need to have `done = assert.async()` simply using the async function will already work correctly.

Reply:
I will add Mocha in separate PR.